schedule/yast/sle/flows/disable_beta.yaml

```diff
@@ -0,0 +1,3 @@
+---
+# In QU/maintenance we disable the beta stuff
+access_beta: []
```

Comment:
we are going to remove this feature, it is just a prototype. Only one overwrite with default schedule is maintainable.

Reply:
ah sorry, forgot to remove that file (was a test, I'm not using it)